src/py_semantic_taxonomy/adapters/routers/router.py

```diff
@@ -12,19 +12,24 @@
 from py_semantic_taxonomy.dependencies import get_graph_service, get_search_service
 from py_semantic_taxonomy.domain import entities as de
 
-router = APIRouter()
+router = APIRouter(prefix="/v1")
```

Comment:
Can we then make the `/api/v1`?

Reply:
Absolutely, we should probably also rename this router to `api_router` or similar.

Reply:
Yeah, had the same thought. Thanks.

Reply:
Fixed in https://github.com/cauldron/py-semantic-taxonomy/commit/6516771b379c99e589d35e1c276c450a3f26c709